src/pages/alternatives.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.choice {
+    font-size: 1.2rem;
+    font-weight: cera-bold;
+    margin-bottom: 0;
+}
+
+.firstQuote {
+    background: $blue;
+    color: $off-white;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+    padding: 2.5em 1em 1em;
+    text-align: center;
+}
+
+.information {
+    margin-bottom: 0;
+    margin-top: 1rem;
+}
+
+.link {
+    color: $blue;
+    margin-bottom: 0;
+}
+
+.quote {
+    margin-bottom: 0;
+}
+
+.secondQuote {
+    background: $teal;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+    padding: 2.5em 1em 1em;
+    text-align: center;
+}
+
+.signature {
+    font-size: 0.68rem;
+    font-style: italic;
+    margin-left: 10rem;
+}
+
+.statPercentage {
+    font-size: 4rem;
+}
+
+.thirdQuote {
```

Comment:
Since each quote item has many of the same shared styles, I'd suggest making a generic `.quote` class, and the adding a second class to modify that. So in scss you'd have:

```scss
.quote {
    background: $teal;
    margin-bottom: 2rem;
    margin-top: 2rem;
    padding: 2.5em 1em 1em;
    text-align: center;
}

.tealBackground {
   background: $teal;
}

.blueBackground {
   background: $blue
}
```

Then, when you want to use them in the markup, you'd do this:
```
<div className={(styles.quote, styles.blueBackground)}>
```
or 
```
<div className={(styles.quote, styles.tealBackground)}>
```